mono/eglib/test/memory.c

```diff
@@ -30,8 +30,112 @@ test_memory_zero_size_allocations (void)
         return OK;
 }
 
+/*
+ * Test the following macros with alignment value as signed int
+ *     <> ALIGN_TO
+ *     <> ALIGN_DOWN_TO
+ *     <> ALIGN_PTR_TO
+ */
+static RESULT
+test_align_signed_int (void)
+{
+	gssize orig_value = 67;
+	gssize result, exp_value;
+	gpointer orig_ptr = (gpointer)orig_value;
+	gpointer result_ptr, exp_ptr;
+
+	// test case #1
+        int align = 1;
+	result = ALIGN_TO(orig_value, align);
```

Comment:
Nit: these should probably be `ALIGN_TO (orig_value, align);`

The indentation in this file also seems to be really wonky, but it was that way beforehand and I can sort it out in a separate PR.

Reply:
Will change the function call format. 

And will try to fix the indentation problem. No guarantee though, cause it doesn't look aligned in my editor. I wonder if there is an editor which has the same indentation mechanism as GitHub? It is really painful to deal with this.

Reply:
The issue is just that it mixes spaces and tabs; converting everything to tabs should solve the problem for the given file. It's hard to get an editor that's going to universally find these issues because whether or not they appear aligned is dependent on how many spaces are used for indentation compared with how many your editor is using per-tab.

Github uses 8-space tabs, and my editor uses 4, so the issue with memory.c was very obvious in my editor. On other files though, it has 4 spaces in the file, and so it looks fine in my editor and broken on Github... :(

Reply:
nvm, git diff actually shows indentation problem

Reply:
It looks like the file is still inconsistent overall, though your diff itself is mostly fixed. There are a ton of trailing spaces after the later  `<>ALIGN_TO`, `<> ALIGN_DOWN_TO`, and `<> ALIGN_PTR_TO` lines for some reason, and lines 109/114 have a trailing tab.

We should probably run a linter for this sort of thing on PR diffs, at least for trailing whitespace - I believe CoreCLR/CoreFX has one already? So maybe in the consolidated repo? I know I've accidentally committed trailing whitespace before and discovered it after the fact :\

Reply:
The format looks pretty good to me now.